extensions/libarchive/MergeContent.cpp

```diff
@@ -342,6 +352,54 @@ std::shared_ptr<core::FlowFile> ZipMerge::merge(core::ProcessContext *context, c
   return flowFile;
 }
 
+void AttributeMerger::mergeAttributes(core::ProcessSession *session, std::shared_ptr<core::FlowFile> &mergeFlow) {
+  for (const auto& pair : getMergedAttributes()) {
+    session->putAttribute(mergeFlow, pair.first, pair.second);
+  }
+}
+
+std::map<std::string, std::string> AttributeMerger::getMergedAttributes() {
+  std::map<std::string, std::string> mergedAttributes;
+  bool isFirst = true;
+  for (const auto& flow : flows_) {
+    if (isFirst) {
+      mergedAttributes = flow->getAttributes();
+      isFirst = false;
+    } else {
+      processFlowFile(flow, mergedAttributes);
+    }
+  }
+  return mergedAttributes;
+}
+
+void KeepOnlymergedAttributesMerger::processFlowFile(const std::shared_ptr<core::FlowFile> &flow, std::map<std::string, std::string>& mergedAttributes) {
```

Comment:
Minor, but the capitalization of `merged` looks off.

Reply:
Good catch, it was a find and replace error :)